docs/execute-notebooks.md

```diff
@@ -43,7 +43,7 @@ Here's an example of adding this tag with a {myst:directive}`code-cell` directiv
 
 ````markdown
 ```{code-cell}
-:tags: raises-exception
+:tag: raises-exception
```

Comment:
Does this work in jupytext?

Reply:
I think jupytext uses tags, rather than tag. Though it does use square brackets for multiple tags. 

https://jupytext.readthedocs.io/en/latest/formats-markdown.html

My feeling is that we should define the syntax we want for MyST, and then upstream the improvements to jupytext to make it work, rather than the other way around

Reply:
I checked by running examples that "tag" works for a single string while "tags" requires brackets in jupytext.

I think it's important to keep documentation in sync with reality as much as possible. I was asked to help, and changing the docs is all I am qualified to do.

Reply:
Can we change this all to `tags: [raises-exception]`, that should work in both myst and jupytext correct?

Reply:
Yes, that works.